etc/kayobe/inventory/group_vars/overcloud

```diff
@@ -0,0 +1,3 @@
+---
+# Use a virtual environment for remote execution of ansible modules.
+ansible_python_interpreter: /opt/kayobe/venvs/kayobe/bin/python
```

Comment:
Do we require a hard-coded path here?

Reply:
Yes, ansible can't cope with variable expansion for this variable.